pkg/scd/constraints_handler.go

```diff
@@ -2,29 +2,350 @@ package scd
 
 import (
 	"context"
+	"database/sql"
+	"fmt"
 
+	"github.com/golang/geo/s2"
 	"github.com/interuss/dss/pkg/api/v1/scdpb"
+	"github.com/interuss/dss/pkg/auth"
 	dsserr "github.com/interuss/dss/pkg/errors"
+	dssmodels "github.com/interuss/dss/pkg/models"
+	scdmodels "github.com/interuss/dss/pkg/scd/models"
+	"github.com/interuss/dss/pkg/scd/repos"
 )
 
 // DeleteConstraintReference deletes a single constraint ref for a given ID at
 // the specified version.
 func (a *Server) DeleteConstraintReference(ctx context.Context, req *scdpb.DeleteConstraintReferenceRequest) (*scdpb.ChangeConstraintReferenceResponse, error) {
-	return nil, dsserr.BadRequest("not yet implemented")
+	// Retrieve Constraint ID
+	idString := req.GetEntityuuid()
+	if idString == "" {
+		return nil, dsserr.BadRequest("missing Constraint ID")
+	}
+	id := scdmodels.ID(idString)
```

Comment:
nit: You could fold this to:
```
id := scdmodels.ID(req.GetEntityUuid())
if len(id) == 0 {
  return ...
}
```

Reply:
Thanks, done.  Also, this was copy-pasted from somewhere else, so I made the change in the other places as well so the 9 places we do this are consistent.  I used id.Empty() to conform with the plurality of the other places and so that hypothetically there might be a slight performance advantage.